### PR TITLE
PDF generation get client timezone

### PR DESCRIPTION
**Pull request by ratheesh-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

**Review by sreehari-aot — APPROVED** [review]
LGTM

File: forms-flow-api/src/formsflow_api/resources/form_process_mapper.py [hunk 1]
```diff
@@ -431,6 +431,7 @@ def get(form_id: string, submission_id: string):
         """PDF generation and rendering method."""
         try:
             if auth.has_one_of_roles([REVIEWER_GROUP, CLIENT_GROUP]):
+                timezone = request.args["timezone"]
```
Comment (sreehari-aot) [inline forms-flow-api/src/formsflow_api/resources/form_process_mapper.py:434]: KeyError if no "timezone" in `request.args`.
Pls make sure `return err.error, err.status_code` wont crash the server

**Review by sreehari-aot — APPROVED** [review]
LGTM!

File: forms-flow-api/src/formsflow_api/resources/form_process_mapper.py (hunk 1 above)
Comment (ratheesh-aot) [reply forms-flow-api/src/formsflow_api/resources/form_process_mapper.py:434]: Okay will update

**Review by abhilash-aot — APPROVED** [review]

**Pull request merged**